### PR TITLE
Added tum logo easter egg

### LINKED ISSUE #203: Easter Eggs!
How did we release three versions without any?!?!?!

### PR DESCRIPTION
**Pull request by TG908**
Tripple tapping on the tum logo changes it from the blue version to the rainbow version

Fixes #203

### CODE REVIEW

File: TUM Campus App/CardViewController.swift [hunk 1]
```diff
@@ -79,10 +80,19 @@ extension CardViewController {
...
         guard let view = nib?.first else { return }
+        logoView = view
         view.frame = CGRect(x: 0, y: 0, width: 100, height: 40)
         self.navigationItem.titleView = view
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(tumLogogTap(sender:)))
+        gestureRecognizer.numberOfTapsRequired = 3
+        view.addGestureRecognizer(gestureRecognizer)
+    }
+    
+    @IBAction func tumLogogTap(sender: AnyObject) {
```
[the remark this answers is not in the document]
Comment (mathiasquintero) [reply TUM Campus App/CardViewController.swift]: Oh, fuck you Sonarqube!

File: TUM Campus App/CardViewController.swift [hunk 2]
```diff
@@ -79,10 +80,19 @@ extension CardViewController {
     
     func setupLogo() {
         let bundle = Bundle.main
-        let nib = bundle.loadNibNamed("TUMLogoView", owner: nil, options: nil)?.flatMap { $0 as? UIView }
+        let nib = bundle.loadNibNamed("TUMLogoView", owner: nil, options: nil)?.flatMap { $0 as? TUMLogoView }
         guard let view = nib?.first else { return }
+        logoView = view
         view.frame = CGRect(x: 0, y: 0, width: 100, height: 40)
         self.navigationItem.titleView = view
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(tumLogogTap(sender:)))
```
[the remark this answers is not in the document]
Comment (TG908) [reply TUM Campus App/CardViewController.swift]: The storyboard is already falling apart (1 FPS) 😩

[the remark this answers is not in the document]
Comment (mathiasquintero) [reply TUM Campus App/CardViewController.swift]: Ah, wait I did this in a .xib. That shouldn't be too complicated. It's only one view

[the remark this answers is not in the document]
Comment (TG908) [reply TUM Campus App/CardViewController.swift]: Im not sure if a GestureRecognizer belongs in the view

[the remark this answers is not in the document]
Comment (mathiasquintero) [reply TUM Campus App/CardViewController.swift]: I know it's not very MVC. But it doesn't feel like something the CardViewController should be taking care of.

**Comment by TG908** [comment]
@mathiasquintero this can be merged now

**Pull request merged**